### PR TITLE
vi-352 Refactor current-clients

### PR DESCRIPTION
**Pull request by emilykim13**
## Summary
- Refactored `current-clients.md` into separate folders per integration
- [Vi-352](https://jira.devops.va.gov/browse/VI-352)

### CODE REVIEW

**Review by bramleyjl — APPROVED** [review]

File: products/identity/Products/Sign-In Service/Identity Team Integrations/current-clients.md
Comment (joeniquette) [inline products/identity/Products/Sign-In Service/Identity Team Integrations/current-clients.md:1]: this is missing the pega, mhv account creation api, arp, and ves integrations. I cant say my list is exhaustive but I have some of them listed in datadog here:
- https://vagov.ddog-gov.com/dashboard/i72-yy8-i97/identity-sts-performance?fromUser=false&refresh_mode=sliding&from_ts=1727897889272&to_ts=1728502689272&live=true
- https://vagov.ddog-gov.com/dashboard/79d-d36-vnf/identity-user-auth-sis-performance?fromUser=false&refresh_mode=sliding&from_ts=1728329892454&to_ts=1728502692454&live=true

**Review by joeniquette — CHANGES_REQUESTED** [review]

**Review by joeniquette — APPROVED** [review]
LGTM...we should have another ticket which updates the pages for those that missing

File: products/identity/Products/Sign-In Service/Identity Team Integrations/current-clients.md
Comment (emilykim13) [reply products/identity/Products/Sign-In Service/Identity Team Integrations/current-clients.md:1]: Oops rerequested review but I just saw this, let me add those

Comment (emilykim13) [reply products/identity/Products/Sign-In Service/Identity Team Integrations/current-clients.md:1]: Nvm I will make a new ticket and add more details for the missing integrations, ty!

Comment (emilykim13) [reply products/identity/Products/Sign-In Service/Identity Team Integrations/current-clients.md:1]: [New ticket](https://jira.devops.va.gov/browse/VI-629)

**Pull request merged**